src/ontology/nbo-idranges.owl

```diff
@@ -59,7 +59,3 @@ EquivalentTo: xsd:integer[> 50000 , <= 59999]
 Datatype: idrange: 12
 Annotations: allocatedto: "Nicole Vasilevsky"
 EquivalentTo: xsd:integer[> 60000 , <= 69999]
-
-Datatype: idrange: 13
-Annotations: allocatedto: "Ditch Townsend"
-EquivalentTo: xsd:integer[> 70000 , <= 79999]
```

Comment:
You should undo this change. You can easily do this by running:

```
git checkout master -- src/ontology/nbo-idranges.owl
```

In your terminal and create a new commit.

Reply:
Totally lost. Sorry.

1. Do I cancel the pull request and start again? I haven't so far:
2. I had to google what 'terminal' was. I was able to pull up a command line thing and when I copied your instruction and right clicked in the command line (which I discovered by trial and error was the way to paste), I got an error (see screenshot).
![Screenshot_20230212_112147](https://user-images.githubusercontent.com/119126471/218308147-ae74bbad-bf51-42c0-855e-f58137abb17b.png)
3. In due course, would I make the new commit from the Terminal command line, or if it works, what will appear by running your instruction?
4. Why did GitHub desktop make me merge a branch master when the update was already done and merged several days ago? I reverted it because I thought it was an error.
5. Why can't I just go to GitHug desktop and revert the revert?
6. Why did I need to use the NBO-edit filename this time, whereas my own filename worked with the idnames?
7. Was I right to revert yesterday's Ditch functional syntax commit? I just guessed I needed to do that - maybe I didn't but it had no consequences anyway?

I appreciate that you want me to learn things, but would it help more for you just to edit this pull request yourself as there seem to be too many issues I need to understand before I can do this safely myself.

Reply:
> Why can't I just go to GitHug desktop and revert the revert?

You can do that too! Actually, it is probably better.

I think you can call the terminal on windows by clicking, in GitHub Desktop, on the "repository" menu ("terminal" submenu). also, you will have to navigate in the terminal to the repository.

I recommend doing this tutorial: https://oboacademy.github.io/obook/tutorial/intro-cli-1/

Reply:
I reverted the revert. I think we can move on.